### PR TITLE
Feature/sc 5172 edita topic

### PR DESCRIPTION
**Pull request by IzaSad**
https://ticketsystem.hpi-schul-cloud.org/browse/SC-5172

### CODE REVIEW

**Review by Grzesiek80 — APPROVED** [review]

**Review by d3n153 — DISMISSED** [review]
Fine from my side.

File: page-objects/pages/coursePages/CRSSAddEditTopicPage.js [hunk 1]
```diff
@@ -94,6 +102,34 @@ async function topicTitleList() {
 	return elementHelpers.getTextFromAllElements(topicSelector);
 }
 
+async function clickOnTopicEditPencilButton(name) {
+	let listOfTopicElements = await driver.$$(topicSelector);
```
Comment (pauljanns) [inline page-objects/pages/coursePages/CRSSAddEditTopicPage.js:106]: I suggest using an elementHelmpers method here. E.g. getListOfAllElements()

**Review by pauljanns — CHANGES_REQUESTED** [review]
One minor change has to be made, afterwards feel free to merge.

File: step_definitions/topic-steps.js [hunk 2]
```diff
@@ -19,3 +19,24 @@ When(/^.* clicks on the topic with name (.*)$/, function (topicName) {
...
+});
+When(/^.* changes topic name (.*)$/, async function (changedTopicName) {
+	await addEditTopicPage.setTopic(changedTopicName);
+});
+When(/^.* finds title (.*) and changes title on (.*) and text (.*) of the topic$/, async function (contentTitle, changedContentTitle, changedDescription) {
+	await addEditTopicPage.findContentByTitleAndChanged(contentTitle, changedContentTitle, changedDescription);
+	await addEditTopicPage.clickCreateTopicButton();
+});
+Then(/^.* should see changed topic with name (.*) and content title (.*) and description (.*) is visible on the topic page$/,async function (changedTopicName, changedContentTitle, changedDescription) {
+	// await addEditTopicPage.(changedTopicName, changedContentTitle, changedDescription);
```
Comment (d3n153) [inline step_definitions/topic-steps.js:33]: Can this be removed, when it's not used?

**Review by d3n153 — COMMENTED** [review]
Found one little thing, which I can also remove with my PR if it's not needed.

**Review by pauljanns — APPROVED** [review]

**Review by d3n153 — DISMISSED** [review]

**Review by pauljanns — APPROVED** [review]

**Review by Grzesiek80 — APPROVED** [review]

**Pull request merged**